db/migrate/20200620163915_rename_lowercase_accounts_index.rb

```diff
@@ -0,0 +1,5 @@
+class RenameLowercaseAccountsIndex < ActiveRecord::Migration[5.2]
+  def change
+    rename_index :accounts, 'index_accounts_on_username_and_domain_lower', 'old_index_accounts_on_username_and_domain_lower'
```

Comment:
I see no reason for this to be separate?

Reply:
> The migration is split in two because adding the index is done with disable_ddl_transaction!, meaning that if the rename were inside it and something happens during the add_index, the renaming would have already happened and re-running the migration would fail in confusing ways.

Reply:
My thinking it's like what a gitlab migration helper method would do for zero-downtime deployment and all of those helper methods happen within one migration. Would definitely be better organization-wise. You could do an `unless index_name_exists?('old_index_accounts_on_username_and_domain_lower')` to prevent failures from re-runs.

Reply:
Alright.